crates/rmf_site_mesh/src/primitives/arrow.rs

```diff
@@ -113,3 +113,48 @@ pub fn flat_arrow_mesh_between(
         ),
     )
 }
+
+pub fn make_triangular_arrow() -> MeshBuffer {
+    let base_positions = vec![
+        [0.0, 0.0, 1.0],  // 0
+        [0.5, 0.0, 0.0],  // 1
+        [-0.5, 0.0, 0.0], // 2
+        [0.0, 0.0, 0.2],  // 3
+        [0.0, 0.3, 0.4],  // 4
+    ];
+
+    let base_indices: Vec<u32> = vec![0, 1, 4, 0, 4, 2, 3, 4, 1, 3, 2, 4, 0, 3, 1, 0, 2, 3];
```

Comment:
I haven't tried it out but if you declared those as `Vec<usize>` instead of `Vec<u32>`, would that allow us to get rid of all `as usize` casts?